### ISSUE TITLE
Feature request: Want TFSlim to automatically download the pre-trained checkpoint

### ISSUE THREAD

**Issue by sfujiwara** [tensorflow/tensorflow#15873]
I want TFSlim to automatically download the pre-trained checkpoint like Keras:
https://keras.io/applications/

When I use Keras, the code below automatically download the pre-trained VGG 16 weights:

```python
from keras.applications.vgg16 import VGG16
from keras.preprocessing import image
from keras.applications.vgg16 import preprocess_input
import numpy as np

model = VGG16(weights='imagenet', include_top=False)
```

On the other hands, TFSlim required to download the checkpoint manually.

Is there any plans to add the feature?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by asimshankar** [comment]
Nope, there are no plans of adding that support AFAIK.
(CCing @sguada in case I'm mistaken)

In general, we'd encourage the use of `tf.keras` where possible.
